libs/blocks/caas-config/caas-config.js

```diff
@@ -673,7 +689,8 @@ const CopyBtn = () => {
 
   const getUrl = () => {
     const url = window.location.href.split('#')[0];
-    return `${url}#${utf8ToB64(JSON.stringify(state))}`;
+    state.fetchCardsFromFloodgateTree = 'default';
+    return `${url}#${utf8ToB64(JSON.stringify(state, fgKeyReplacer))}`;
```

Comment:
Could we use the delete operator here and avoid the additional replacer function call?

```
delete state.fetchCardsFromFloodgateTree;
return `${url}#${utf8ToB64(JSON.stringify(state))}`;
```

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/delete

May be @chrischrischris or someone else can confirm as I am not fully aware of the side effects of using it.

Reply:
@sukamat I didn't delete it because every time Authors click on "Copy Link", they will loose their FG checkbox selection in the configurator, causing the card results on the Configurator page to show Default cards instead of Floodgate cards. Hence deleting it from the JSON while performing copy link. If we prefer this option, will update this. Will discuss this with you.